docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: Database Access with Redis
+description: How to configure Teleport Database Access with Redis.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Redis is available starting from Teleport `9.0`.
+</Details>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Configure mutual TLS authentication between Teleport and your Redis database.
+- Connect to Redis database through Teleport.
+
+## Prerequisites
+
+- Teleport version `9.0` or newer.
+- Redis database (standalone or cluster) version `6.0` or newer.
+- `redis-cli` installed and added to your system's `PATH` environment variable.
+
+<Admonition type="note" title="Note">
+  Redis `7.0` and RESP3 (REdis Serialization Protocol) are currently not supported.
+</Admonition>
+
+## Step 1/5. Set up Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/5. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/5. Create Redis users.
+
+Redis user must be protected by a strong password. We recommend using a password generator built-in in Redis.
+To generate password connect to a Redis instance and call:
+
+```
+ACL GENPASS
+```
+
+or use OpenSSL if you have it installed on your machine
+
+```shell
+openssl rand -hex 32
+```
+
+This command generates a password like that `0a49c95a25b0f92787d07a5b6bdc1a25b3a2130a1b338ec524f1b5dcd708bc65`.
+
+Create `users.acl` file with users. File defines Redis users, passwords required to log in as a given user
+and sets of Redis ACL rules. Redis allows providing passwords as a plain text or SHA256 checksum version.
+We strongly recommend to use SHA256 checksum instead of plain text password.
+
```

Comment:
I think we should give an example of a command to use for creating an SHA256 checksum of a password.

Reply:
Added